proto/hummock.proto

```diff
@@ -41,13 +41,27 @@ message InputLevel {
   repeated SstableInfo table_infos = 3;
 }
 
-message LevelDelta {
+message IntraLevelDelta {
   uint32 level_idx = 1;
   uint64 l0_sub_level_id = 2;
   repeated uint64 removed_table_ids = 3;
   repeated SstableInfo inserted_table_infos = 4;
 }
 
+message GroupConstruct {
+  CompactionConfig group_config = 1;
+}
+
+message GroupDestroy {}
+
+message LevelDelta {
```

Comment:
Naming it `LevelDelta` is a bit confusing since `GroupConstruct` and `GrounpDestroy` have nothing to do with level. How about naming it `GroupDelta` or `GroupVersionDelta` instead?

Reply:
> Naming it `LevelDelta` is a bit confusing since `GroupConstruct` and `GrounpDestroy` have nothing to do with level. How about naming it `GroupDelta` or `GroupVersionDelta` instead?

ok